containers/osbuild-composer/entrypoint.py

```diff
@@ -241,6 +244,20 @@ def _spawn_worker():
             stderr=subprocess.STDOUT,
         )
 
+    @staticmethod
+    def _wait_for_syslog(addr):
+        print("Waiting for fluentd syslog server", file=sys.stderr)
+        for _ in range(Cli.SYSLOG_RETRIES):
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            result = sock.connect_ex(addr)
```

Comment:
This can raise exceptions. The `_ex` variant only silences exceptions on the `connect(2)` level but not any of the other calls done. Since you're not exiting otherwise they need to be caught (things like `getaddrinfo` failing, etc).

Reply:
I switched to socket.connect() so that different error are handled differently and unexpected errors are reported.